File: Stack/Node.cs
using System;

namespace Stack
{
	public class Node
	{
		private object _data;
		public Node Next;
		
		public Node(object data)
		{
			_data = data;
			Next = null;
		}
		
		public object Data()
		{
			return _data;
		}
		
		public override string ToString()
		{
			return _data.ToString() + " -> " +
				(Next != null? Next.ToString() : "END");
		}
	}
}

File: Queue/Node.java

public class Node 
{
	public Node Next;
	public Node Last;
	public Object Data;
	
	public Node(Object data)
	{
		Data = data;
		Next = null;
		Last = null;
	}
	
	public String toString()
	{
		return Data.toString() + " -> " +
	            (Next != null ? Next.toString() : "END");
	}
}
